src/pages/token-auth-rules/primitive-rules/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Primitive Rules
+metaTitle: Token Auth Rules - Primitive Rules
+description: Documentation of the available rules.
+---
+
+## Additional Signer
+An additional account must sign this transaction.
+
+### Fields
+* **address** - The address that must sign the Transaction
+
+## Amount
+The amount of tokens being transferred is compared (greater than, less than, or equal to) against an amount.
+
+### Fields
+* **amount** - The amount to be compared against
+* **operator** - The comparison operation to use: greater than, less than, equal to
+* **field** - The payload field to compare against
+
+## Namespace
```

Comment:
The `Namespace` section is missing description.  I think this special type of rule warrants a good description as its fairly complex to understand.